docs/.dockerignore

```diff
@@ -10,9 +10,8 @@
 build/
 Debug/
 Release/
-circuits/cpp/build/
-circuits/cpp/build-wasm/
-circuits/cpp/build-coverage/
+# Eventually remove these once people don't have circuits artifacts around
+circuits/cpp/build*
```

Comment:
good call

Reply:
Just realized this is docs dockerignore. Probably really moot point. Ill remove this actually